### PR TITLE
JCLOUDS-1095: Update OpenStack KeyStone Support for Zaqar

### PR DESCRIPTION
**Pull request by fribeiro1**

### CODE REVIEW

File: openstack-marconi/src/main/java/org/jclouds/openstack/marconi/v1/MarconiApiMetadata.java [hunk 1]
```diff
@@ -58,7 +58,7 @@ protected MarconiApiMetadata(Builder builder) {
 
    public static Properties defaultProperties() {
       Properties properties = BaseHttpApiMetadata.defaultProperties();
-      properties.setProperty(SERVICE_TYPE, ServiceType.QUEUES);
+      properties.setProperty(SERVICE_TYPE, ServiceType.MESSAGING);
```
Comment (nacx) [inline openstack-marconi/src/main/java/org/jclouds/openstack/marconi/v1/MarconiApiMetadata.java]: Agree to update the service type to the latest version. Not on the scope of this PR, but would it make sense to rename the provider to "openstack-messaging"? @zack-shoylev thoughts?

Comment (fribeiro1) [inline openstack-marconi/src/main/java/org/jclouds/openstack/marconi/v1/MarconiApiMetadata.java]: It certainly would, as well as other changes (e.g. change package name), but I have left that to another sub-task.

**Comment by nacx** [comment]
As soon as build [#3390](https://jclouds.ci.cloudbees.com/job/jclouds/3390/) finishes, the new snapshots will be available and this build should succeed.

**Comment by nacx** [comment]
Snapshots should be in place now: rebuild please!

**Comment by nacx** [comment]
@fribeiro1 now the unit tests are failing due to the change. Mind fixing them?

**Comment by fribeiro1** [comment]
@nacx The unit tests here, not in the core project, right?

**Comment by nacx** [comment]
Right. If you look at the links in the PR summary and click in the "Details" one you'll see the build output.

**Comment by fribeiro1** [comment]
@nacx Found them, will work on them today.

**Comment by fribeiro1** [comment]
@nacx Fixed in https://github.com/jclouds/jclouds/pull/939

**Comment by nacx** [comment]
Thanks! Next time, please just add the commit to the PR, as both are part of the same change. (There's no need to do it now; I'll squash the commits before merging).

**Comment by nacx** [comment]
Hmmm I see now fixes for the tests are in the main repo. Dismiss the comment.

**Comment by nacx** [comment]
Tests pass now locally. Pushed to master as [042680d9](http://git-wip-us.apache.org/repos/asf/jclouds-labs-openstack/commit/042680d9). Thanks @fribeiro1!

**Pull request closed without merging**